scripts/harmonize.py

```diff
@@ -133,7 +142,28 @@ def __init__(self, chr, pos, ref, alt, af, filt, an):
         self.an = int(an)
 
 
-def harmonize(file_in, file_ref, chr_col, pos_col, ref_col, alt_col, af_col, beta_col, require_gnomad, passing_only, gnomad_min_an, gnomad_max_abs_diff, pre_aligned):
+def print_best_match(variant_list: List[VariantData], require_gnomad: bool, gnomad_max_abs_diff: float, keep_best_duplicate: bool):
```

Comment:
looks fine otherwise but would change this to choose_best_variant which returns it and then you can do whatever you want with it later.... i.e. separate the logic from representation